android/src/main/java/com/pritesh/calldetection/CallDetectionManagerModule.java

```diff
@@ -120,13 +120,13 @@ public void phoneCallStateUpdated(int state) {
             case TelephonyManager.CALL_STATE_OFFHOOK:
                 //Device call state: Off-hook. At least one call exists that is dialing, active, or on hold, and no calls are ringing or waiting.
                 wasAppInOffHook = true;
-                jsModule.callStateUpdated("Offhook");
+                jsModule.callStateUpdated("Offhook", null);
                 break;
             //Incoming
             case TelephonyManager.CALL_STATE_RINGING:
                 // Device call state: Ringing. A new call arrived and is ringing or waiting. In the latter case, another call is already active.
-                jsModule.callStateUpdated("Incoming");
+                jsModule.callStateUpdated("Incoming", incomingNumber);
```

Comment:
Instead `incomingNumber` you should have `phoneNumber`

Reply:
Oh right, thanks!